frontend/src/app/modules/metric-finder/models/test-result.ts

```diff
@@ -24,16 +24,20 @@ export class TestInfo implements TestInfoDTO {
   }
```

Comment:
just from a convention point of view, is there a reason this file is not called test-result.model.ts ?